src/styles/index.scss

```diff
@@ -1,3 +1,85 @@
+@import 'variables';
+
+// Body styles
```

Comment:
The comments is unnecessary here.

```suggestion
```

Reply:
Thanks a lot for recommending. I'll take it into account in my future deploys.